src/massive/munit/TestResult.hx

```diff
@@ -97,7 +102,19 @@ class TestResult
 	/**
 	 * Class constructor.
 	 */
-	public function new() {}
+	public function new() 
+	{
+		passed = false;
+		executionTime = 0.0;
+		name = "";
+		className = "";
+		description = "";
+		async = false;
+		ignore = false;
+		args = null;
+		error = null;
+		failure = null;
```

Comment:
107-116, Why duplicate the initialization of these variables?

Reply:
It looks like this was unneeded part of the merge -- in Srikanth's fork these are initialized only in the constructor, perhaps to make the class easier to extend... or just an older version that was fixed in a different way?

https://github.com/sganapavarapu1/MassiveUnit/blob/master/src/massive/munit/TestResult.hx#L42